content/en/docs/concepts/storage/volumes.md

```diff
@@ -1034,6 +1034,11 @@ As a result, operators do not have to make any
 configuration changes to existing Storage Classes, PersistentVolumes or PersistentVolumeClaims
 (referring to in-tree plugins) when transitioning to a CSI driver that supersedes an in-tree plugin.
 
+{{< note >}}
+* The existing PV created by in-tree volume plugin can still be used in the future without any configuration changes even after the `CSIMigration` is completed. 
+* A new PV with a manifest that uses in-tree volume plugins can still be created, even after the in-tree volume plugin is removed from Kubernetes.
```

Comment:
nit: After CSI migration effort completes and a corresponding in-tree volum plugin is removed, operations on PVs are actually done by CSI drivers, so how about `refers to` instead of `(a manifest that) uses (in-tree volume plugins ...)`.

another nit: a single PV cannot refer to multiple in-tree volume plugins, so it is better to use singular/plural cases consistently, like `new PVs refering to in-tree volume plugins`.

Reply:
Thank you for you kind suggestion! I've corrected the second sentence as you pointed out.